src/modules/companies/examples/example-response-for-get-company-by-registration-number.json

```diff
@@ -0,0 +1,44 @@
+{
+    "companiesHouseRegistrationNumber": "00000001",
+    "companyName": "TEST COMPANY LTD",
+    "registeredAddress": {
+        "addressLine1": "1 Test Street",
+        "locality": "Test City",
+        "postalCode": "A1 2BC",
+        "country": "United Kingdom"
+    },
+    "industries": [
+        {
+            "class":  {
+                "code": "59112",
+                "name": "Video production activities"
+            },
+            "code": "1009",
+            "name": "Information and communication"
```

Comment:
I made a slight change to the response format to match DTFS and mean that less mapping is needed there.